integrationtests/tests/tests.rs

```diff
@@ -142,6 +142,21 @@ async fn peg_outs_are_only_allowed_once_per_epoch() -> Result<()> {
     .await
 }
 
+#[tokio::test(flavor = "multi_thread")]
+async fn peg_ins_that_are_unconfirmed_are_rejected() -> Result<()> {
+    test(2, |_fed, user, bitcoin, _, _| async move {
+        let peg_in_address = user.client.get_new_pegin_address(rng()).await;
+        let (proof, tx) = bitcoin.send_and_mine_block(&peg_in_address, Amount::from_sat(10000));
+        let result = user.client.peg_in(proof, tx, rng()).await;
+
+        // TODO make return error more useful
+        assert!(result.is_err());
```

Comment:
The `result` is an error because no epochs are run, so the federation doesn't know about the new blocks yet I assume?

Reply:
Right and enough blocks didn't pass.